packages/glob/src/internal-path-helper.ts

```diff
@@ -1,5 +1,5 @@
-import * as assert from 'assert'
 import * as path from 'path'
+import assert from 'assert'
```

Comment:
reaction due to moving `"esModuleInterop": true` to the root `tsconfig.json` - which previously was only defined in `packages/github/tsconfig.json`